clear-lake-restoration-app/src/Components/pages/lakeTChain/LakeTchain.js

```diff
@@ -186,7 +186,6 @@ export default function LakeTchain(props) {
                 m.push([new Date(element.DateTime_UTC).getTime(),2,parseFloat(element["Height_2m"])]);
                 m.push([new Date(element.DateTime_UTC).getTime(),3,parseFloat(element["Height_3m"])]);
                 m.push([new Date(element.DateTime_UTC).getTime(),4,parseFloat(element["Height_4m"])]);
-                
```

Comment:
Are these the most recent changes?

Reply:
clear-lake-restoration-app/src/Components/pages/lakeTChain/LakeTchain.js

that and this

Reply:
Well that and I added the parentheses around the h-1 to the code in this: 

let values =  (((j-(h-1))/(heightM-(h-1))) * (parseFloat(element["Height_surface"]) - parseFloat(element["Height_11m"])) + parseFloat(element["Height_11m"]));

because I saw the white space that Valentina mentioned for UA01 and I figured that was why.

Reply:
there's still blanks NR-02
![image](https://user-images.githubusercontent.com/42662469/168216091-de412969-272e-4dd3-b30d-299a61c4bf06.png)

Reply:
When you hover over the blank, what is it saying? is it still saying height 8m, 0 degrees?

Reply:
@KennethLieu how does the changes look on your computer? I see the same thing as Alison

Reply:
have you been testing the examples and cases we have been supplying you with through this PR?